client/webserver/webserver.go

```diff
@@ -226,27 +226,39 @@ func New(core clientCore, addr string, logger dex.Logger, reloadHTML bool) (*Web
 	// api endpoints
 	mux.Route("/api", func(r chi.Router) {
 		r.Use(middleware.AllowContentType("application/json"))
-		r.Post("/getfee", s.apiGetFee)
-		r.Post("/newwallet", s.apiNewWallet)
-		r.Post("/openwallet", s.apiOpenWallet)
-		r.Post("/closewallet", s.apiCloseWallet)
-		r.Post("/register", s.apiRegister)
 		r.Post("/init", s.apiInit)
-		r.Post("/login", s.apiLogin)
-		r.Post("/withdraw", s.apiWithdraw)
+
+		// TODO: Allow register page to not require /user and /defaultwalletcfg
+		// until authorized with a cookie. These currently must be accessible to
+		// set the app password on the browser.
```

Comment:
This is unfortunate.  The browser frontend will need to be modified so that /user and /defaultwalletcfg are not required to set the app pass on the initial setup screen.  For /user, at least work without the wallet and order data.